consts.php

```diff
@@ -15,270 +15,11 @@
 /** Repositories that either don't have a composer type or have a weird composer type, though we still want installer */
 const FORCE_INSTALLER_REPOS = [
     'silverstripe-behat-extension',
-    'silverstripe-serve',
-    // recipe-solr-search doesn't include recipe-cms or recipe-core unlike our other recipes
-    'recipe-solr-search',
-    // recipe-blog requires a theme for the unit tests to work, the config and dependency for which are
-    // supplied by installer. Just adding a theme as a dev-dependency is insufficient because we'd still
-    // lack the yml config to activate that theme
-    'recipe-blog',
```

Comment:
serve is no longer supported, and we have no reason to run CI on the recipes anymore.